packages/core/src/Services/logout.js

```diff
@@ -14,6 +14,8 @@ const doLogout = response => {
     localStorage.removeItem('closed_toast_notifications');
     localStorage.removeItem('is_wallet_migration_modal_closed');
     localStorage.removeItem('active_wallet_loginid');
+    localStorage.removeItem('verification_code.system_email_change');
```

Comment:
how about  request_email?

Reply:
Checking this

Reply:
I think we shouldn't remove items from localstore here as I tired for request_email and for some reason its failing. Investigating deeper into this.